docs/fsharp-cheatsheet.md

```diff
@@ -67,88 +68,88 @@ Other common examples are `F` or `f` for 32-bit floating-point numbers, `M` or `
 
     let s, f, d, bi = 4.14F, 4.14, 0.7833M, 9999I
 
-	// [fsi:val s : float32 = 4.14f]
-	// [fsi:val f : float = 4.14]
-	// [fsi:val d : decimal = 0.7833M]
-	// [fsi:val bi : System.Numerics.BigInteger = 9999]
+    // [fsi:val s : float32 = 4.14f]
+    // [fsi:val f : float = 4.14]
+    // [fsi:val d : decimal = 0.7833M]
+    // [fsi:val bi : System.Numerics.BigInteger = 9999]
 
 See [Literals (MSDN)](http://msdn.microsoft.com/en-us/library/dd233193.aspx) for complete reference.
 
 <a name="Functions"></a>Functions
 ---------
-The `let` keyword also defines named functions.
+The `let` keyword also defines named functions:
 
-	let negate x = x * -1 
-	let square x = x * x 
-	let print x = printfn "The number is: %d" x
+    let negate x = x * -1 
+    let square x = x * x 
+    let print x = printfn "The number is: %d" x
 
     let squareNegateThenPrint x = 
-		print (negate (square x)) 
+        print (negate (square x)) 
 
 ### Pipe and composition operators
-Pipe operator `|>` is used to chain functions and arguments together. Double-backtick identifiers are handy to improve readability especially in unit testing:
+Pipe operator `|>` is used to chain functions and arguments together. Double-backtick identifiers are handy to create a symbol with spaces in it to improve readability, especially for unit testing:
 
-	let ``square, negate, then print`` x = 
-		x |> square |> negate |> print
-
-This operator is essential in assisting the F# type checker by providing type information before use:
+    let ``square, negate, then print`` x = 
+        x |> square |> negate |> print
+	
+This operator [what operator?] is essential in assisting the F# type checker by providing type information before use:
 
     let sumOfLengths (xs : string []) = 
-		xs 
-		|> Array.map (fun s -> s.Length)
-		|> Array.sum
+        xs 
+        |> Array.map (fun s -> s.Length)
+        |> Array.sum
 
-Composition operator `>>` is used to compose functions:
+Composition operator `>>` is used to compose functions in a way that's more more concise than shown above. Also, appending a backtick to a symbol is a conventional way to mark the new symbol as a variant of a previously defined symbol. It's like appending `1` but cooler:
 
-	let squareNegateThenPrint' = 
-		square >> negate >> print
+    let squareNegateThenPrint' = 
+        square >> negate >> print
   
 ### Recursive functions
 The `rec` keyword is used together with the `let` keyword to define a recursive function:
 
-	let rec fact x =
-	    if x < 1 then 1
-	    else x * fact (x - 1)
+    let rec fact x =
+        if x < 1 then 1
+        else x * fact (x - 1)
 
-*Mutually recursive* functions (those functions which call each other) are indicated by `and` keyword:
+*Mutually recursive* functions (functions that call each other) are indicated by `and` keyword:
 
-	let rec even x =
-	   if x = 0 then true 
-	   else odd (x - 1)
+    let rec even x =
```

Comment:
needs a better example - v distracting to read a nonsenensical function compared to doing a `% 2 = 0` check